src/coreclr/tests/src/dirs.proj

```diff
@@ -7,19 +7,14 @@
     </PropertyGroup>
 
     <ItemGroup>
-      <DisabledProjects Include="$(TestRoot)TestWrappers*\**\*.csproj" />
       <DisabledProjects Include="$(TestRoot)*\**\cs_template.csproj" />
-      <DisabledProjects Include="$(TestRoot)Common\Coreclr.TestWrapper\Coreclr.TestWrapper.csproj" />
```

Comment:
What happened to these projects? How are they disabled with the change now?

Reply:
I am adding a global blocking clause for everything under Common in this change, I believe that to be far easier than manually disabling each test. Frankly speaking I'm not super happy about the Common name, I was thinking about renaming it to something like "eng", but I think it's a bit of busywork that can easily wait.